extension/source/QuillPage/TimeCell.ts

```diff
@@ -0,0 +1,17 @@
+import { IReadableCell } from '../cells/ICell';
+import MemoryCell from '../cells/MemoryCell';
+import delay from '../helpers/delay';
+
+export default function TimeCell(accuracy: number): IReadableCell<number> {
+  const cell = new MemoryCell(Math.floor(Date.now() / accuracy));
+
+  (async () => {
+    while (true) {
+      const now = Date.now();
+      await delay(accuracy * Math.ceil(now / accuracy) - now);
+      await cell.write(accuracy * Math.round(Date.now() / accuracy));
+    }
+  })();
```

Comment:
It doesn't lol. Time never stops right?

Yeah it's not ideal 😄. I think we can address this later though since this is only used in the demo. I'm thinking about making a specialized cell for time+accuracy which will update lazily in a similar way to a FormulaCell to avoid this problem.

Reply:
Will be addressed in followup work, not huge risk ATM so leaving as is in this work.